src/main/java/edu/harvard/iq/dataverse/DatasetVersionUI.java

```diff
@@ -458,6 +458,10 @@ public void setMetadataValueBlocks(DatasetVersion datasetVersion) {
                         mdb.setEmpty(false);
                         datasetFieldsForView.add(dsf);
                     }
+                    //Setting Local Display on Create on mdb when there are any set at dataverse level 
+                    if ( dsf.getDatasetFieldType().getLocalDisplayOnCreate() != null ){
+                       mdb.setLocalDisplayOnCreate(dsf.getDatasetFieldType().getLocalDisplayOnCreate());
```

Comment:
If there's one with displayoncreate true and a later one with false, this sets the block displayonCreate back to false? 

Also not seeing how the fieldtype localdisplayoncreate can be set at this point (versus always being null).

Reply:
Could be null if there are no custom settings on the block. Point taken on false overriding true. Will fix.